pkg/model/project.go

```diff
@@ -26,8 +26,87 @@ import (
 
 var (
 	githubScopes = []string{"read:org"}
+
+	builtinAdminRBACRole = &ProjectRBACRole{
+		Name:      builtinRBACRoleAdmin.String(),
+		Policies:  builtinAdminRBACPolicy,
+		IsBuiltin: true,
+	}
+	builtinAdminRBACPolicy = []*ProjectRBACPolicy{
+		{
+			Resources: []*ProjectRBACResource{
+				{
+					Type: ProjectRBACResource_ALL,
+				},
+			},
+			Actions: []ProjectRBACPolicy_Action{
+				ProjectRBACPolicy_ALL,
+			},
+		},
+	}
+	builtinEditorRBACRole = &ProjectRBACRole{
+		Name:      builtinRBACRoleEditor.String(),
+		Policies:  builtinEditorRBACPolicy,
+		IsBuiltin: true,
+	}
+	builtinEditorRBACPolicy = []*ProjectRBACPolicy{
+		{
+			Resources: []*ProjectRBACResource{
+				{Type: ProjectRBACResource_APPLICATION},
+				{Type: ProjectRBACResource_DEPLOYMENT},
+				{Type: ProjectRBACResource_EVENT},
+				{Type: ProjectRBACResource_DEPLOYMENT_CHAIN},
+				{Type: ProjectRBACResource_INSIGHT},
+			},
+			Actions: []ProjectRBACPolicy_Action{
+				ProjectRBACPolicy_ALL,
```

Comment:
There is no DELETE action for those kinds of resources so to be safe let's specify the actions explicitly with excluding the DELETE.